tiledb/sm/filesystem/test/unit_uri.cc

```diff
@@ -184,44 +184,61 @@ TEST_CASE("URI: Test schemes", "[uri]") {
   CHECK(URI("tiledb://namespace/array").is_tiledb());
 }
 
-TEST_CASE("URI: Test REST components", "[uri]") {
+TEST_CASE("URI: Test REST components, valid", "[uri]") {
```

Comment:
Can you add tags for these two tests?

Reply:
They have the same tags the original tests had.